feature highlights

juliapradel opened the pull request:
Bitte Korrektur lesen und unten ergänzen

Commit:
Update README.md

dbosen commented on README.md:
```diff
@@ -10,6 +10,40 @@ Description
+* **Create articles dynamically with paragraphs**: Using paragraphs, you can add text, pictures, videos, Instagram or Twitter cards to your article – and change the order by dragging and dropping the content wherever you like it. Thanks to a preview mode, you get a good idea of how your article will look like in the end.
+* **Working in distributed teams**: Thunder is a web-based system, so you can add and edit articles from everywhere. All you need is a computer and access to the Internet. Every modification to an article is saved in revisions. So, you can easily understand who did changes to your article. Plus, you can share a link to an unpublished article with external co-workers and give them access for a lim ...
+* **User management**: In Thunder, you can define lots of different roles for every need—and allocate various rights.
+* **Liveblog**: This module helps to cover events as they happen, like sport events or elections.
```
Its not yet integrated, and will not be released before drupal 8.3 is released in April. So maybe we should not have it in the featurelist as of now.

dbosen commented on README.md:
```diff
@@ -10,6 +10,40 @@ Description
+* **Arrange your content**: Use categories and tags to sort your articles. Plus, you can easily create lists of content, such as the most read articles, or a package about a specific topic.
+
+NOCH OFFEN
+* Feeds: ????
```
leave out, not an important feature

dbosen commented on README.md:
```diff
@@ -10,6 +10,40 @@ Description
+
+NOCH OFFEN
+* Feeds: ????
+* Multilingual: ???
```
That means, that you can have a web page in multiple languages. E.G. a german and an english version.

juliapradel replied:
Buttons; automatisch auf deutsche Seite kommen; Texte verlinkt

dbosen commented on README.md:
```diff
@@ -10,6 +10,40 @@ Description
+  * Migration support
+  * Big community
+    * Support system
+  * Rest API/ decoupled frontend
```
I would leave this out of the core features.

dbosen commented on README.md:
```diff
@@ -10,6 +10,40 @@ Description
+  * verschiedene Ansichten (Teaser)
+* Technical details
+  * Scalability
+    * Caching system
```
Caching system can be left out

dbosen commented on README.md:
```diff
@@ -10,6 +10,40 @@ Description
+* Multilingual: ???
+* Structure articles
+  * WYSIWYG-Editor
+  * verschiedene Ansichten (Teaser)
```
Teaser system is out of scope for feature highlights

dbosen commented on README.md:
```diff
@@ -10,6 +10,40 @@ Description
+* Technical details
+  * Scalability
+    * Caching system
+  * Migration support
```
Also out of scope

dbosen commented on README.md:
```diff
@@ -10,6 +10,40 @@ Description
+  * Rest API/ decoupled frontend
+  * Themeable
+  * Extendable with 2500+ modules
+  * Tested with 10.000+ automated tests
```
Tests are out of scope

Commit:
Update README.md

deleted Liveblog and the details Daniel mentioned (feeds, teaser, caching system, migration support, rest api, tests)

added multilingual, flexible system, and big drupal community

dbosen committed:
Merge branch 'master' into feature-highlights

dbosen committed:
Reduce highlights
